face_detector/CMakeLists.txt

```diff
@@ -70,18 +70,27 @@ install(DIRECTORY launch param
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/
 )
 
-# if(CATKIN_ENABLE_TESTING)
-#   find_package(rostest REQUIRED)
+if(CATKIN_ENABLE_TESTING)
+  find_package(rostest REQUIRED)
 
-#   catkin_download_test_data(${PROJECT_NAME}_face_detector_noface_test_diamondback.bag
-#     http://download.ros.org/data/face_detector/face_detector_noface_test_diamondback.bag
-#     DESTINATION ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/test
-#     MD5 37f043be780a4511c853379defdd9855)
-#   catkin_download_test_data(${PROJECT_NAME}_ace_detector_withface_test_diamondback.bag
-#     http://download.ros.org/data/face_detector/face_detector_withface_test_diamondback.bag
-#     DESTINATION ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/test
-#     MD5 59126117e049e69d577b7ee27251a6f8)
+  catkin_download_test_data(${PROJECT_NAME}_noface_test_diamondback.bag
+    http://download.ros.org/data/face_detector/face_detector_noface_test_diamondback.bag
+    DESTINATION ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/test
+    MD5 37f043be780a4511c853379defdd9855)
+  catkin_download_test_data(${PROJECT_NAME}_withface_test_diamondback.bag
+    http://download.ros.org/data/face_detector/face_detector_withface_test_diamondback.bag
+    DESTINATION ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION}/test
+    MD5 9210704c6cd88862a8e6e2b4a5407e27)
```

Comment:
I don't know why but checksum of the downloaded file was different, so I had to change this.